File: python_MVC/entity/Filed.py
# entity/filed.py

class Filed:
    def __init__(self, size):
        self.stack = [0] * size  # スタックのサイズを指定
        self.sp = 0  # スタックポインタ

    def getStack(self):
        return self.stack

    def getSp(self):
        return self.sp

    def setSp(self, sp):
        self.sp = sp

File: java_MVC/entity/Filed.java
package entity;

public class Filed {
    private int[] stackArray;
    private int sp; // スタックポインタ

    public Filed(int size) {
        stackArray = new int[size];
        sp = 0;
    }

    public int[] getStack() {
        return stackArray;
    }

    public int getSp() {
        return sp;
    }

    public void setSp(int sp) {
        this.sp = sp;
    }
}
